docs/website/contents/for-developers/Glossary.md

```diff
@@ -5,7 +5,7 @@ Notes on the use and maintenance of this glossary:
 - Definitions are preceded by a colon (`:`), eg `:Slot` to make it easy to find the using `Ctrl-F`.
 - When adding a new definition, add a colon in front of it. If the definition has synonyms, separate them by commas (prefixing a colon for each synonym).
 
-## :Active Slot Coefficient, :f
+## : Active Slot Coefficient, :f
```

Comment:
> Also separate the headers in Glossary from the :, otherwise the right side navigation bar omits the first word

Hmm, these changes are unfortunate, as the colon was intentionally set that way:

> - Definitions are preceded by a colon (`:`), eg `:Slot` to make it easy to find the using `Ctrl-F`.
> - When adding a new definition, add a colon in front of it. If the definition has synonyms, separate them by commas (prefixing a colon for each synonym).

A `;` seems to not cause this weird behavior, maybe change `:` to `;` everywhere?

Reply:
Done!